README.md

```diff
@@ -20,12 +23,19 @@ Positions are not final. We are happy to revisit positions as new information be
 - `under consideration` - WebKit's position on this specification is being worked out and is not yet settled.
 - `not considering` - WebKit does not intend to look at this specification in the near future.
 
+Note that positions on this repository do not reflect implementation status. We might like something we do not get around to implement for one reason or another. And we might dislike something we nevertheless have to implement.
+
 #### Other possible issue states
 
 - `invalid` - The issue is not about a specification which would be implemented in a browser engine such as WebKit.
 - `duplicate` - The issue is a duplicate of another issue.
 
-### Filter issues
+### Resolving an issue with a position
 
-- [All issues with a position](https://github.com/WebKit/standards-positions/issues?q=is%3Aopen+is%3Aissue+label%3A%22position%3A+neutral%22%2C%22position%3A+not+considering%22%2C%22position%3A+oppose%22%2C%22position%3A+support%22%2C%22position%3A+under+consideration%22).
-- [All issues without a position](https://github.com/WebKit/standards-positions/issues?q=is%3Aopen+is%3Aissue+-label%3A%22position%3A+neutral%22+-label%3A%22position%3A+not+considering%22+-label%3A%22position%3A+oppose%22+-label%3A%22position%3A+support%22+-label%3A%22position%3A+under+consideration%22+-label%3Ameta).
+Reviewing proposals is always encouraged and as such please leave feedback and add relevant labels to issues.
+
+To add a "position: _X_" label where _X_ is one of the positions documented above, please follow this simple process so there is opportunity for WebKit-at-large to reflect and chime in as needed. As WebKit consists of many individuals across companies and timezones some asynchronicity is desirable.
```

Comment:
do we want any sort of comment about what we should do if this process isn't followed? given that's happened on several occasions, due to not everyone being 100% up to speed with what the process is? (and I'm sure not everyone is gonna notice this in the readme!)

Reply:
I think given that we're open to new information and revisiting our position it's okay. Hopefully over time everyone will try to take this new process into account. Maybe if it still happens a year from now we should revisit?